packages/js/eslint-plugin/src/configs/vue.ts

```diff
@@ -23,7 +23,7 @@ export = {
     "vue/block-order": [
       "error",
       {
-        order: ["script[setup]", "template", "script:not([setup])", "style"],
+        order: ["script[setup]", "script:not([setup])", "template", "style"],
```

Comment:
This is the only change that was actually up for review, right? LGTM!

Reply:
Yes! Thank you for highlighting it